app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,8 @@
 	<%= stylesheet_link_tag    "application", media: "all" %>
 	<%= javascript_include_tag "application" %>
 	<%= csrf_meta_tags %>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+  <script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
```

Comment:
I don't think you should need this to include jquery - that's in there by default.  jqueryui I think makes sense to add here though.